site2/docs/txn-use.md

```diff
@@ -34,6 +34,12 @@ This section provides an example of how to use the transaction API to send and r
       ```
       acknowledgmentAtBatchIndexLevelEnabled=true
       ```
+    If you want to enable deduplication for transaction, follow the steps below.
```

Comment:
It's a required configuration, without message deduplication, we can't support exactly-once semantic.
@Anonymitaet Please help refine this one. And the `acknowledgmentAtBatchIndexLevelEnabled` also required or optional @congbobo184

Reply:
Thanks @codelipenghui 

Currently, @momo-jun is more focused on doc work and I’m more focused on doc-related PIP work
so @momo-jun , could you please take a look at this? Thanks